handbook/engineering.md

```diff
@@ -257,7 +257,7 @@ UX is key for any software. APIs that take longer than 500ms to respond can caus
 
 In the ideal case, the count query will be covered by an index and be extremely fast. In the worst case, the query will be counting filtering on calculated data, which results in a full (multi) table scan on big tables.
 
-One way to solve this is to pre-calculate data in an async fashion, so we would have a cron that once every hour or so would count whatever we want counted, store the counts, and then counting things is as fast as reading a row in a table.
+One way to solve this is to pre-calculate data in an async fashion, so we would have a cron that once every hour or so would count whatever we want counted, store the counts, and then counting things is as fast as reading a row in a table. 
```

Comment:
This paragraph is all one sentence and I couldn't decipher the meaning to rewrite it. Please rewrite in simple terms for clarity.

Reply:
```suggestion
An approach we've taken to addressing this is pre-calculating aggregations and counts that take a long time to generate. By generating these results beforehand and storing them, we can return results by reading a single row from a table when the information is needed.
```